sensing/gnss_poser/include/gnss_poser/convert.hpp

```diff
@@ -100,6 +100,41 @@ GNSSStat NavSatFix2UTM(
   return utm;
 }
 
+bool utm_origin_needs_init = true;
+GNSSStat utm_origin;
+GNSSStat utm_local;
```

Comment:
It is not used anywhere else in the code, it is only used in the `NavSatFix2LocalCartesianUTM` function. It should be moved into the function scope. If you are worried about runtime performance, there is not much difference because of the [return value optimization performed by the compilers](https://stackoverflow.com/questions/12953127/what-are-copy-elision-and-return-value-optimization).

Reply:
And pushed also.